.github/actions/archive-artifacts/action.yaml

```diff
@@ -42,3 +42,22 @@ runs:
             sh -c "kubectl exec -i -n ${NAMESPACE} kcat -- \
                        kcat -L -b ${KAFKA_SERVICE} -t {}  -C -o beginning -e -q -J \
                        > /tmp/artifacts/data/${STAGE}/kafka-messages-{}.log"
+    - name: Dump MongoDB
+      shell: bash
+      continue-on-error: true
+      run: |-
+        set -exu
+
+        ZENKO_MONGODB_DATABASE="${ZENKO_MONGODB_DATABASE:-zenko-database}"
+        MONGODB_ROOT_USERNAME="${MONGODB_ROOT_USERNAME:-root}"
+        MONGODB_ROOT_PASSWORD="${MONGODB_ROOT_PASSWORD:-rootpass}"
+        NAMESPACE="${NAMESPACE:-default}"
+
+        MONGODB_COLLECTIONS=$(kubectl exec -n ${NAMESPACE} data-db-mongodb-sharded-mongos-0 -- mongo ${ZENKO_MONGODB_DATABASE} -u ${MONGODB_ROOT_USERNAME} -p ${MONGODB_ROOT_PASSWORD} --authenticationDatabase admin --quiet --eval "db.getCollectionNames().join(' ')")
+
+        for collection in ${MONGODB_COLLECTIONS}; do
```

Comment:
do we want to keep only the special collections?
For now, I see ~400 collections kept (due to object lock or test failing), they all have less than 10 objects anyway

Reply:
not sure what we want to do...

it takes ~1min20, which is significant (maybe not after 2h of test, but we need to reduce this 2h anyway...)

- we should try to reduce the time. Probably a large time is related to kubectl kube setup & I/O transfers out of the container... May be much faster using mongo dump ; and possibly "copying" the file from the container after the tool has finished, or creating a new container with a host-volume to directly "save" the file to the host...
- if we can't make it significantly faster, we may need some filtering indeed AND/OR do this only if the previous job has failed...